src/backend/columnar/columnar_customscan.c

```diff
@@ -61,11 +71,19 @@ static Cost ColumnarIndexScanAddStartupCost(RelOptInfo *rel, Oid relationId,
 											IndexPath *indexPath);
 static Cost ColumnarIndexScanAddTotalCost(PlannerInfo *root, RelOptInfo *rel,
 										  Oid relationId, IndexPath *indexPath);
-static void RecostColumnarSeqPath(RelOptInfo *rel, Oid relationId, Path *path);
+static void CostColumnarSeqPath(RelOptInfo *rel, Oid relationId, Path *path);
```

Comment:
then I think we should rename:
RecostColumnarPaths -> CostColumnarPaths
RecostColumnarIndexPath -> CostColumnarIndexPath

Reply:
Let me do that as part of another refactor.